### DISCUSSION TITLE
汉化后游戏问题

### DISCUSSION THREAD

**Discussion by mikepig** [GpointChen/FFXIVChnTextPatch-GP#288] [Q&A]
打完高清汉化包后游戏突然打不开了，说是dx11问题。不打高清字体就没有问题。求助
An unexpected error has occurred. Exiting Final Fantasy XIV
ffxiv_dd11exe+506E7B

**Comment by GpointChen** [comment]
和 #278 相同問題

需要更改設置為高解析度

![](https://user-images.githubusercontent.com/34565875/215534898-4893ef01-5876-414c-9ea5-4062bbf88396.png)
